app/controllers/katello/remote_execution_controller.rb

```diff
@@ -28,11 +30,13 @@ def prepare_composer
       end
 
       def hosts
-        if params[:scoped_search].present?
-          params[:scoped_search]
-        else
-          ::Host.where(:id => params[:host_ids].try(:split, ','))
-        end
+        find_bulk_hosts('edit_hosts',
+          included: {
+            ids: params[:host_ids],
```

Comment:
Found a bug. It expects host_ids to be an array like ```:host_ids =>[1,2,3]```, while what gets sent to remote command controller is something like ```:host_ids =>"1,2,3"``` so need to change this to ```params[:host_ids].try(:split, ',')```

Reply:
may be do something like ```params[:host_ids].is_a?(Array) ?  params[:host_ids]: params[:host_ids].try(:split, ',')```

Reply:
@jlsherrill would you think something like this might be better in ```Concerns::Api::V2::BulkHostsExtensions```

Reply:
no, the browser should be sending an array IMO

Reply:
hmm too many places to fix that then imo.